linebot/models/flex_message.py

```diff
@@ -325,26 +426,50 @@ def __init__(self, url=None, margin=None, size=None, aspect_ratio=None, **kwargs
             and the previous component in the parent box
         :param str size: Maximum size of the icon width
         :param str aspect_ratio: Aspect ratio of the icon
+        :param str position: Reference position for placing this box
```

Comment:
[nits]
The order of parameters in docstring is not the same as that in the method definition